src/translations/data.json

```diff
@@ -723,1200 +723,1246 @@
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "Знах.",
+    "be": "Він.",
+    "pl": "Bier.",
+    "cs": "4.pád",
+    "sk": "Aku.",
+    "sl": "Tož.",
+    "hr": "Aku.",
+    "sr": "Аку.",
+    "mk": "Аку.",
     "bg": "Вин."
   },
   "caseGen": {
     "en": "Gen.",
     "isv": "Rod.",
     "ru": "Род.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "Род.",
+    "be": "Род.",
+    "pl": "Dop.",
+    "cs": "2.pád",
+    "sk": "Gen.",
+    "sl": "Rod.",
+    "hr": "Gen.",
+    "sr": "Ген.",
+    "mk": "Ген.",
     "bg": "Род."
   },
   "caseLoc": {
     "en": "Loc.",
     "isv": "Měst.",
-    "ru": "Мест.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "ru": "Пред.",
+    "uk": "Місц.",
+    "be": "Мес.",
+    "pl": "Miej.",
+    "cs": "6.pád",
+    "sk": "Lok.",
+    "sl": "Mest.",
+    "hr": "Lok.",
+    "sr": "Лок.",
+    "mk": "Лок.",
     "bg": "Мест."
   },
   "caseDat": {
     "en": "Dat.",
     "isv": "Dat.",
     "ru": "Дат.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "Дав.",
+    "be": "Дав.",
+    "pl": "Cel.",
+    "cs": "3.pád",
+    "sk": "Dat.",
+    "sl": "Daj.",
+    "hr": "Dat.",
+    "sr": "Дат.",
+    "mk": "Дат.",
     "bg": "Дат."
   },
   "caseIns": {
     "en": "Ins.",
     "isv": "Tvor.",
     "ru": "Твор.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "Оруд.",
+    "be": "Твор.",
+    "pl": "Narz.",
+    "cs": "7.pád",
+    "sk": "Inš",
+    "sl": "Orod.",
+    "hr": "Ins.",
+    "sr": "Инс.",
+    "mk": "Инс.",
     "bg": "Твор."
   },
   "caseVoc": {
     "en": "Voc.",
     "isv": "Zvat.",
     "ru": "Зват.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "Клич.",
+    "be": "Кліч.",
+    "pl": "Woł.",
+    "cs": "5.pád",
+    "sk": "Vok.",
+    "sl": "Zval.",
+    "hr": "Vok.",
+    "sr": "Вок.",
+    "mk": "Вок.",
     "bg": "Зват."
   },
   "degreesOfComparison": {
     "en": "Degrees of comparison",
     "isv": "Stupnje sravnenja",
     "ru": "Степени сравнения",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "Ступені порівняння",
+    "be": "Ступені параўнання",
+    "pl": "Stopnie porówniania",
+    "cs": "Stupně srovnání",
+    "sk": "Stupne porovnania",
+    "sl": "Stopnje primerjave",
+    "hr": "Stupnjevi usporedbe",
+    "sr": "Степени поређења",
+    "mk": "Степени на споредување",
     "bg": "Степени за сравнение"
   },
   "positive": {
```

Comment:
in slovak (grammatical context) I encountered mostly the terms "pozitív" "komparatív" and "superlatív" respectively

Reply:
let's change

Reply:
I changed